jupyter/nbgrader/unit-tests/grader/unit-tests/data.csv

```diff
@@ -0,0 +1,144 @@
+1, "AI for Marketing","2019-08-01","2019-12-31"
```

Comment:
Is this file used? I couldn't find any references in the PR

Reply:
It's not used anywhere yet, but I plan to use it for future tests to make sure that we can reference external files in our notebooks.

Reply:
Sounds good! I'll remove it for now then.